roles/analysis/tasks/analysis-c2r.yml

```diff
@@ -6,6 +6,9 @@
     convert2rhel analyze
     {{ analysis_convert2rhel_preconv_opts }}
     {{ analysis_convert2rhel_enable_repos_args }}
+    {% if analysis_convert2rhel_eus %}
+    --eus
+    {% endif %}
```

Comment:
We will need to add this to the conversion task too.

Reply:
Will do.